react/Modal/index.jsx

```diff
@@ -9,9 +9,94 @@ import migrateProps from '../helpers/migrateProps'
 import palette from '../../stylus/settings/palette.json'
 import Portal from 'preact-portal'
 
-const ModalContent = ({ children, className }) => (
-  <div className={cx(styles['c-modal-content'], className)}>{children}</div>
-)
+class ModalContent extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { displayGhostHeader: false }
+  }
+
+  componentDidMount() {
+    this.scrollingContent.addEventListener('scroll', this.handleScroll, {
```

Comment:
The event listener should be removed in `componentWillUnmount`.

Reply:
Indeed.